site/reference/cli/fn/eval/README.md

```diff
@@ -107,7 +107,9 @@ fn-args:
   Allowed values: stdout|unwrap|<OUT_DIR_PATH>
   1. stdout: output resources are wrapped in ResourceList and written to stdout.
   2. unwrap: output resources are written to stdout, in multi-object yaml format.
-  3. OUT_DIR_PATH: output resources are written to provided directory, the directory is created if it doesn't already exist.
+  3. OUT_DIR_PATH: output resources are written to provided directory.
```

Comment:
hmm... I am still trying to come up with the right wording to convey. From user's perspective, what is happening is that: files containing output resources are getting overwritten (or created). existing files not containing the output resources are not deleted.

As I typed this, I realized it's not about files containing output resources, it is files targeted by the output resources. One more thing I realized, if a resource moved from file f1 to file f2 between two runs of `fn eval/render`, both files will exist with duplicate resources.

I am slowly warming up to the idea of `pruning by default`

Reply:
I though we settled on not silently accumulating cruft in the output dir.

Reply:
I haven't implemented yet. This is to document the current behavior. I can implement when we decide upon the behavior.